specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.1/TextAnalytics.json

```diff
@@ -0,0 +1,1346 @@
+    },
+    "SentenceSentiment": {
+      "type": "object",
+      "required": [
+        "text",
+        "sentiment",
+        "confidenceScores",
+        "offset",
+        "length",
+        "aspects",
+        "opinions"
+      ],
+      "properties": {
+        "text": {
+          "type": "string",
+          "description": "The sentence text."
+        },
+        "sentiment": {
+          "type": "string",
+          "description": "The predicted Sentiment for the sentence.",
+          "enum": [
+            "positive",
+            "neutral",
+            "negative"
+          ],
+          "x-ms-enum": {
+            "name": "SentenceSentimentValue",
+            "modelAsString": false
+          }
+        },
+        "confidenceScores": {
+          "description": "The sentiment confidence score between 0 and 1 for the sentence for all classes.",
+          "$ref": "#/definitions/SentimentConfidenceScorePerLabel"
+        },
+        "offset": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The sentence offset from the start of the document."
+        },
+        "length": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The length of the sentence by Unicode standard."
+        },
+        "aspects": {
+          "type": "array",
+          "description": "The array of aspect object for the sentence.",
+          "items": {
+            "$ref": "#/definitions/SentenceAspect"
+          }
+        },
+        "opinions": {
+          "type": "array",
+          "description": "The array of opinion object for the sentence.",
+          "items": {
+            "$ref": "#/definitions/SentenceOpinion"
+          }
+        },
+        "warnings": {
+          "type": "array",
+          "description": "The warnings generated for the sentence.",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "SentenceAspect": {
+      "type": "object",
+      "required": [ "confidenceScores", "id", "length", "offset", "relations", "sentiment", "text" ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Unique, non-empty document identifier."
+        },
+        "sentiment": {
+          "type": "string",
+          "enum": [ "positive", "mixed", "negative" ],
+          "description": "Aspect level sentiment for the aspect in the sentence."
+        },
+        "confidenceScores": {
+          "description": "Aspect level sentiment confidence scores for the aspect in the sentence.",
+          "$ref": "#/definitions/AspectConfidenceScoreLabel"
+        },
+        "offset": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The aspect offset from the start of the sentence."
+        },
+        "length": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The length of the aspect by Unicode standard."
```

Comment:
Same comment.

Reply:
Okay, i removed "Unicode" in the Swagger for V3.1-Preview.